api/src/test/java/io/strimzi/api/kafka/model/KafkaCrdIT.java

```diff
@@ -154,6 +155,38 @@ public void testKafkaWithInvalidJmxAuthentication() {
                 containsStringIgnoringCase("spec.kafka.jmxOptions.authentication.type: Unsupported value: \"not-right\": supported values: \"password\"")));
     }
 
+    @Test
+    void testJmxOptionsWithoutRequiredOutputDefinitionKeys() {
+        Throwable exception = assertThrows(
+            KubeClusterException.InvalidResource.class,
+            () -> {
+                createDelete(Kafka.class, "JmxTrans-output-definition-with-missing-required-property.yaml");
+            });
+
+        String expectedString1 = "spec.kafka.jmxOptions.jmxTrans.outputDefinitions.outputType in body is required";
+        String expectedString2 = "spec.kafka.jmxOptions.jmxTrans.outputDefinitions.name in body is required";
```

Comment:
in other parts of the codebase, we removed the usage of these variables (recently) which sounds useless. Maybe you can do the same here.